src/main/scala/io/iohk/ethereum/utils/ByteUtils.scala

```diff
@@ -70,21 +85,115 @@ object ByteUtils {
   }
 
 
-  def bytesToInts(bytes: Array[Byte]): Array[Int] = {
-    val buffer =  ByteBuffer.wrap(bytes).order(ByteOrder.LITTLE_ENDIAN)
-    val ints = new Array[Int](bytes.length / 4)
-    buffer.asIntBuffer().get(ints)
-    ints
+  def bytesToInts(bytes: Array[Byte], bigEndian: Boolean): Array[Int] = {
+    val ret = new Array[Int](bytes.length / 4)
+    bytesToIntsMut(bytes, ret, bigEndian)
+    ret
   }
 
-  def intsToBytes(ints: Array[Int]): Array[Byte] = {
-    val buf = ByteBuffer.allocate(ints.length * 4).order(ByteOrder.LITTLE_ENDIAN)
-    buf.asIntBuffer().put(ints)
-    buf.array()
+  def intsToBytes(ints: Array[Int], bigEndian: Boolean): Array[Byte] = {
```

Comment:
you have almost everywhere `bigEndian` set to false, maybe you could add setting it to false?

Reply:
to be honest i would rather have the user of this function to make decision about which conversions should be used